jupyterhubutils/lsst_configure.py

```diff
@@ -0,0 +1,39 @@
+from .authenticator import (LSSTGitHubOAuthenticator,
+                            LSSTCILogonOAuthenticator, LSSTJWTAuthenticator)
+from .spawner import LSSTSpawner
+
+
+def lsst_configure(config):
+    '''Do all the LSST-specific configuration based on the authenticator
+    type and environment variables.
+    '''
+    config.spawner_class = LSSTSpawner
+    authclass = None
+    authtype = config.authenticator_type
+    if authtype == 'jwt':
+        authclass = LSSTJWTAuthenticator
+    elif authtype == 'cilogon':
+        authclass = LSSTCILogonOAuthenticator
+    else:
+        authclass = LSSTGitHubOAuthenticator
+    config.authenticator_class = authclass
+    authclass.oauth_callback_url = config.oauth_callback_url
+    if authtype == 'jwt':
+        authclass.signing_certificate = '/opt/jwt/signing-certificate.pem'
+        authclass.username_claim_field = 'uid'
```

Comment:
maybe just make these members on LSSTJWTAuthenticator (and they don't change)

Reply:
Except for signing cert